runtime/autoload/netrw.vim

```diff
@@ -311,7 +311,7 @@ endif
 " Default values for netrw's global variables {{{2
 " Cygwin Detection ------- {{{3
 if !exists("g:netrw_cygwin")
- if has("win32") || has("win95") || has("win64") || has("win16")
+ if has("win32") || has("win64")
   if  has("win32unix") && &shell =~ '\%(\<bash\>\|\<zsh\>\)\%(\.exe\)\=$'
    let g:netrw_cygwin= 1
   else
```

Comment:
(This comment is out of scope of this PR, but ...)
This Cygwin detection is completely broken.
`has("win32")` never returns 1 on Cygwin. So, `g:netrw_cygwin` will be always set to 0 (even on Cygwin).